.github/workflows/test.yml

```
@@ -125,14 +113,11 @@
             ${{ runner.os }}-pods-
 
       - run: bun install
-      
-      - run: bun install
-        working-directory: example
-      
+
       - run: pod install
-        working-directory: example/ios
+        working-directory: apps/example/ios
       
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)